src/shared/pages/welcomePage/script.js

```diff
@@ -0,0 +1,238 @@
+  shortcutsList.appendChild(p);
+  if (launchBrowser.shortcut) {
+    let launchBrowserHotkey = launchBrowser.shortcut;
+
+    // convert Ctrl to Cmd on Mac and command to cmd on Mac
+    if ((await browser.runtime.getPlatformInfo()).os === "mac") {
+      launchBrowserHotkey = launchBrowserHotkey.replace("Ctrl", "CMD");
+      launchBrowserHotkey = launchBrowserHotkey.replace("Command", "CMD");
+    }
+
+    p.id = "launch-browser-shortcut";
+    p.innerText = browser.i18n.getMessage("welcomePageYesShortcutTo", [
+      launchBrowserHotkey.toUpperCase(),
+      await getExternalBrowser(),
+    ]);
+  } else {
+    replaceMessage(p, "welcomePageNoShortcutsFirefox", "addons://shortcuts/shortcuts");
+
+    // remove the manage shortcuts text
+    const manageShortcutsText = document.querySelector(
+      "[data-locale='welcomePageManageShortcuts']"
+    );
+    manageShortcutsText.remove();
+  }
+}
+
+/**
+ * Check the hotkeys for the browser launch and let the user know if
+ * they are not set.
+ *
+ * Also add a link to the chromium shortcuts page.
+ */
+export async function checkChromiumHotkeys() {
+  const hotkeys = await browser.commands.getAll();
+  const launchBrowser = hotkeys.find(
+    (hotkey) => hotkey.name === "launchBrowser"
+  );
+  const launchFirefoxPrivate = hotkeys.find(
+    (hotkey) => hotkey.name === "launchFirefoxPrivate"
+  );
+
+  const shortcutsList = document.getElementById("shortcuts-list");
+
+  // if there are no shortcuts, display a message
+  if (!launchBrowser.shortcut && !launchFirefoxPrivate.shortcut) {
+    const preamble = document.createElement("p");
+    shortcutsList.appendChild(preamble);
+    replaceMessage(
+      preamble,
+      "welcomePageNoShortcutsChromium",
+      "chrome://extensions/shortcuts"
+    );
+
+    // remove the manage shortcuts text
+    const manageShortcutsText = document.querySelector(
+      "[data-locale='welcomePageManageShortcuts']"
+    );
+    manageShortcutsText.remove();
+    return;
+  }
+
+  // Display a message for the Firefox shortcut
+  const span = document.createElement("span");
+  span.classList.add("shortcut-item");
+  if (launchBrowser.shortcut) {
+    const launchBrowserHotkey = launchBrowser.shortcut;
+    span.innerText =
+      browser.i18n.getMessage("welcomePageYesShortcutTo", [
+        launchBrowserHotkey.toUpperCase(),
+        await getExternalBrowser(),
+      ]) + "\n";
+  } else {
+    span.innerText =
+      browser.i18n.getMessage("welcomePageNoShortcutTo", ["Firefox"]) + "\n";
+  }
+  shortcutsList.appendChild(span);
+
+  // Display a message for the Firefox private shortcut
+  const span2 = document.createElement("span");
+  span2.classList.add("shortcut-item");
+  if (launchFirefoxPrivate.shortcut) {
+    const launchFirefoxPrivateHotkey = launchFirefoxPrivate.shortcut;
+    span2.innerText =
+      browser.i18n.getMessage("welcomePageYesShortcutTo", [
+        launchFirefoxPrivateHotkey.toUpperCase(),
+        "Firefox private browsing",
```

Comment:
This text shouldn't be hard-coded. 

You should have a separate string just for `$hotkey$ launches your current page in Firefox private browsing`.

Note that you're also inconsistent, as in other places "Private" is uppercase.

Reply:
Never mind, it was fixed here https://github.com/mozilla/firefox-launch/commit/69cb5a14323021d0441c7027132e43b91da60e10